core/ui/SideBar/Open.tid

```diff
@@ -23,10 +23,20 @@ $button$
 </$droppable>
 \end
 
+\define open-tab-close-tiddler-button()
+<$button message="tm-close-tiddler" tooltip={{$:/language/Buttons/Close/Hint}} aria-label={{$:/language/Buttons/Close/Caption}} class="tc-btn-invisible tc-btn-mini tc-small-gap-right">{{$:/core/images/close-button}}</$button>
+\end
+
+<!-- Override one or both of the following 2 macros with a global macro -->
+<!-- of the same name if you need to change how elements are displayed  -->
```

Comment:
Here I think it's cleaner to have a single comment with a couple of cleanups:

```
<!--
Override one or both of the following two macros with a global or local macro of the same name to change how list items are displayed  -->
```

Reply:
My mistake, of course here it only makes sense to override these macros globally.